src/layers/microscopy-viewer-layer/microscopy-viewer-layer-base.js

```diff
@@ -36,15 +36,25 @@ export class MicroscopyViewerLayerBase extends BaseTileLayer {
     const minZoom = Math.floor(
       -1 * Math.log2(Math.max(props.imageHeight, props.imageWidth))
     );
-    const { sliderValues, colorValues } = props;
+    const { sliderValues, colorValues, channelsOn } = props;
+    const inputColorValues = {};
+    const inputSliderValues = {};
+    Object.keys(colorValues).forEach(channel => {
+      inputColorValues[channel] = channelsOn[channel]
```

Comment:
capture this in a const and use it in both places.
```suggestion
      const channelIsOn = channelsOn[channel];
      inputColorValues[channel] = channelIsOn
```